File: EnforcerWeb/src/app/model/Component.ts
export class Component {
	public component_id: number;
	public project_id: number;
	public name: string;

	constructor( obj: Object = {}) {
		Object.assign(this, obj) ;
	}
 
	public toJSON() {
		return {
			component_id: this.component_id,
			project_id: this.project_id,
			name: this.name,
		}
	}
}

File: EnforcerServices/src/main/java/com/webforged/enforcer/management/data/Component.java
package com.webforged.enforcer.management.data;

import org.springframework.data.annotation.Id;
import org.springframework.data.relational.core.mapping.Table;

import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor
@Table("components")
public class Component {
	
	@Id
	public Long component_id ;
	public Long project_id ;
	public String name;
	
	public Component() {
	}
}